python/cpp/pycategory.cpp

```diff
@@ -20,9 +20,9 @@
 #include <stdio.h>
 #include <exception>
 #include <stdexcept>
-#include "NVCategory.h"
-#include "NVStrings.h"
-#include "util.h"
+#include <nvstrings/NVCategory.h>
+#include <nvstrings/NVStrings.h>
+#include <nvstrings/util.h>
```

Comment:
@davidwendt I had to move `util.h` into `include` in order to make these include statements work properly. Is this `util.h` needed in these files?

Reply:
Please don't change or move this file. 
I don't want this file to be in the include directory and the quotes should not be replaced here.
Was there some reason you needed to do this?

Reply:
Answered below and we can continue discussion there.